### DISCUSSION TITLE
isValid is incorrect when browser autofills the form

### DISCUSSION THREAD

**Discussion by goppi0** [react-hook-form/react-hook-form#7392] [Q&A]
Hi,

My login page uses a simple email/password form and it sets the focus automatically on the email field. When the browser however can autofill the fields, the isValid formState shows false until I change the focus to anything else. Is there a correct way to validate the form after the browser autofilled it?

Thanks,
Goppi

**Comment by bluebill1049** [comment]
i don't think it's possible with browser autofill but you can try to use trigger() at the useEffect?
